.github/workflows/sample-publish.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish samples to NPM
+
+on:
+  push:
+    tags:
+      - 'sample-*'
```

Comment:
I doubt these samples will ever see a version bump or be needed to be integrated with the rest of the projects CI. Creating a new tag just for the sample package to trigger this workflow seems to be the simplest way to get `npm publish` to run for the plugin sample package only. 

@JamieSlome thoughts?